src/components/ReportActionItem/ActionItemButtons.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {View} from 'react-native';
+import Button from '@components/Button';
+import useThemeStyles from '@hooks/useThemeStyles';
+
+type ActionItem = {
```

Comment:
I think `ActionItem` is a bit confusing, since we also have the `ReportActionItem` naming convention (people might think that it's a type of report action.

Can we call this component `ActionableItemButtons`, and call the type `ActionableItem`?

Reply:
Better yet, let's add a comment somewhere explaining what this component does.